qa-tests-backend/src/test/groovy/SACTest.groovy

```diff
@@ -62,8 +63,10 @@ class SACTest extends BaseSpecification {
     static final private Integer WAIT_FOR_VIOLATION_TIMEOUT = isRaceBuild() ? 600 : 60
 
     def setupSpec() {
-        // ROX-6260: pre scan the image to avoid missing risk score
-        Services.scanImage(TEST_IMAGE)
+        // ROX-6260: pre scan the image to avoid missing risk score.
```

Comment:
ROX-6260 is closed now and didn't give me any extra valuable info. I'd hence suggest to remove the reference; feel free to keep (or expand) the comment though.